src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberMethodSymbol.cs

```diff
@@ -235,6 +246,8 @@ private void MethodChecks(MethodDeclarationSyntax syntax, Binder withTypeParamsB
                 else if (!IsStatic)
                 {
                     diagnostics.Add(ErrorCode.ERR_BadExtensionMeth, location);
+                    // PROTOTYPE(readonly-ref): to match PEMethodSymbol, should we SetIsExtensionMethod(false)? See #20282 for more details
```

Comment:
>PROTOTYPE(readonly-ref): [](start = 23, length = 24)

Please remove the prototype prefix, I do not think this is related to the new feature.